chains/v18/chains_dev.json

```diff
@@ -6511,7 +6511,7 @@
                 "type": "orml",
                 "icon": "https://raw.githubusercontent.com/novasamatech/nova-utils/master/icons/tokens/white/vMANTA.svg",
                 "typeExtras": {
-                    "currencyIdScale": "0x0908",
+                    "currencyIdScale": "0x0a08",
```

Comment:
check vManta once again, it's correct. It should be `vToken2: 8` not `vsToken2: 8`